site/content/docs/4.3/examples/navbars/index.html

```diff
@@ -32,7 +32,7 @@
           </div>
         </li>
       </ul>
-      <form class="form-inline my-2 my-md-0">
+      <form class="my-2 my-md-0">
```

Comment:
Wondering if we still need the `my-2` on these forms? Feels like we're jumping the height on them too much on xs/sm viewports.

Reply:
I've now added a margin between the navigation and the form element and removed the additional bottom margin which was set here.